package.json

```diff
@@ -2,7 +2,7 @@
   "author": "James R. Carr <james.r.carr@gmail.com> (http://blog.james-carr.org)",
   "name": "mongoosastic",
   "description": "A mongoose plugin that indexes models into elastic search",
-  "version": "4.5.1",
+  "version": "4.6.0",
```

Comment:
this is a major version IMO

Reply:
Totally agree, this will be 5.0. but we can change it later before publishing.